AppLovin MAX Demo App - Java/app/build.gradle

```diff
@@ -36,7 +38,7 @@ android {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.aar'])
```

Comment:
What exactly does this change do? Should we add it to the Kotlin demo app as well?

Reply:
It doesn't really do anything, but it felt like a good practice. The old one basically says "treat any jars in the libs folder as a dependency". The change expands it include aars. We don't have a libs folder, so it does nothing, but it is handy to be able to drop an aar in there and have it just work.